book/src/bindings/README.md

```diff
@@ -0,0 +1,9 @@
+# Bindings
+
+PRQL has bindings for many languages. These include:
+
+- [Java](./java.md)
+- [JavaScript](./javascript.md)
+- [Python](./python.md)
+- [R](./r.md)
+- [Rust](./rust.md)
```

Comment:
@kasvith if you want to add Elixir here, that would be great!

Reply:
Sure, I was looking into the docs 
Will add elixir

Reply:
@max-sixty https://github.com/PRQL/prql/pull/1576